src/cli/key/sev/create.rs

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: Apache-2.0
+
+use std::fmt::Debug;
+use std::fs::File;
+use std::io::prelude::*;
+use std::io::stdout;
+
+use camino::Utf8PathBuf;
+use clap::Args;
+use elliptic_curve::SecretKey;
+use p384::ecdsa::SigningKey;
+use p384::pkcs8::LineEnding;
+use p384::{elliptic_curve, pkcs8, NistP384};
+use pkcs8::EncodePrivateKey;
+
+/// Generate an SEV key for use with Enarx.
+#[derive(Args, Debug)]
+pub struct Options {
+    /// File path to write SEV key in PEM form
+    #[clap(long)]
+    out: Option<Utf8PathBuf>,
+}
+
+impl Options {
+    pub fn execute(self) -> anyhow::Result<()> {
+        let rng = rand::thread_rng();
+        let signing_key = SigningKey::random(rng);
+
+        // somehow SigningKey does not implement EncodePrivateKey
```

Comment:
Is this an issue with the p384 crate? Has it been reported?

Reply:
filed https://github.com/RustCrypto/signatures/issues/519